AZ3166/src/libraries/AzureIoT/src/DevKitMQTTClient.cpp

```diff
@@ -345,28 +352,23 @@ static bool SendEventOnce(EVENT_INSTANCE *event)
     uint64_t start_ms = SystemTickCounterRead();
 
     CheckConnection();
-    wait_ms(1000);
```

Comment:
Why remove the delay？

Reply:
no delay needed here, should be rubbish code here